StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/Helpers/OnSetterFunctionCalledMessageHandler.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  OnSetterFunctionCalledMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/9/24.
+//
+
+import Foundation
+
+struct OnSetterMessage<Values: Codable & Equatable>: Decodable {
```

Comment:
Any reason this isn't nested in `OnSetterFunctionCalledMessageHandler`?

Also, similar nit as above with naming it "Payload" as opposed to "Message"